# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../anjuta/no-pkgconfig-list-all.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Do not source all the .pc and header files - we will pull in a lot of
dependencies and the list would not match the installed one anyway ...

  - Rene Rebe <rene@exactcode.de>

--- anjuta-2.0.0/global-tags/create_global_tags.sh.orig	2005-03-11 18:53:15.000000000 +0100
+++ anjuta-2.0.0/global-tags/create_global_tags.sh	2005-05-20 11:19:23.000000000 +0200
@@ -66,7 +66,7 @@
 
 # pkg-config libraries
 PKG_CONFIG=`which pkg-config 2>/dev/null`
-if ( [ ! -z $PKG_CONFIG ] && [ -x $PKG_CONFIG ] ) ; then
+if false ; then
     
     PKG_DEFAULT_CONFIG_PATH=`echo $PKG_CONFIG | sed -e 's/\/bin\/pkg-config/\/lib\/pkgconfig'/`
     PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$PKG_DEFAULT_CONFIG_PATH"
